boards/st/stm32mp257f_dk/stm32mp257f_dk_stm32mp257fxx_m33.dts

```diff
@@ -91,3 +91,17 @@
 	current-speed = <115200>;
 	status = "okay";
 };
+
+&i2c8 {
+	/* Pin conflict with i3c4. Enable only with i3c4 disabled */
+	pinctrl-0 = <&i2c8_scl_pz4 &i2c8_sda_pz3>;
+	pinctrl-names = "default";
+	status = "disabled";
+};
+
+&i3c4 {
+	/* Pin conflict with i2c8. Enable only with i2c8 disabled */
+	pinctrl-0 = <&i2c8_scl_pz4 &i2c8_sda_pz3>;
```

Comment:
This is the same as for `&i2c8` - is that intended, or perhaps should have a different AF to route those pins to i3c vs i2c?

Sorry, I don't know this SoC that well, so might be routed to the same AF for both peripherals

Reply:
:-) we missed that. Thanks.
I3C busses have their own pin muxing and properties.
```suggestion
	pinctrl-0 = <&i3c4_scl_pz4 &i3c4_sda_pz3>;
```